.github/CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # Adding/changing the data in repositories.yaml can have large downstream
 # effects, so we're a little stricter on who can sign off on changes here.
-repositories.yaml @jklukas @relud @whd @cmharlow
+repositories.yaml @jklukas @relud @chutten @whd @cmharlow
 
 # The exclusion list in git_scraper.py can cause similar problems (see e.g.
 # https://bugzilla.mozilla.org/show_bug.cgi?id=1745771)
-probe_scraper/scrapers/git_scraper.py @jklukas @relud @whd @cmharlow
+probe_scraper/scrapers/git_scraper.py @jklukas @relud @chutten @whd @cmharlow
```

Comment:
Any particular reason I'm being added to the middle?

Reply:
end of the list of DE, before the list of DSRE